modules/common/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/UaParserEnrichmentSpec.scala

```diff
@@ -51,7 +51,7 @@ class UaParserEnrichmentSpec extends Specification with DataTables {
         "jsonschema",
         SchemaVer.Full(1, 0, 0)
       ),
-      json"""{"useragentFamily":"Safari","useragentMajor":"8","useragentMinor":"0","useragentPatch":null,"useragentVersion":"Safari 8.0","osFamily":"Mac OS X","osMajor":"10","osMinor":"10","osPatch":null,"osPatchMinor":null,"osVersion":"Mac OS X 10.10","deviceFamily":"Other"}"""
+      json"""{"useragentFamily":"Safari","useragentMajor":"8","useragentMinor":"0","useragentPatch":null,"useragentVersion":"Safari 8.0","osFamily":"Mac OS X","osMajor":"10","osMinor":"10","osPatch":null,"osPatchMinor":null,"osVersion":"Mac OS X 10.10","deviceFamily":"Mac"}"""
```

Comment:
As of https://github.com/ua-parser/uap-core/pull/459/files when OS X is detected in the user agent string device family is set to Mac